javalanglib/src/main/scala/java/lang/StringBuilder.scala

```diff
@@ -106,8 +106,8 @@ class StringBuilder
     val oldContent = content
     if (offset < 0 || offset > oldContent.length)
       throw new StringIndexOutOfBoundsException(offset)
-    content =
-      oldContent.substring(0, offset) + str + oldContent.substring(offset)
+    content = oldContent.substring(0, offset) + str + oldContent.substring(
+        offset)
```

Comment:
Huh? Why is this happening? I have seen scalafmt do the opposite (correctly IMHO).